sys/net/gnrc/transport_layer/tcp/gnrc_tcp.c

```diff
@@ -103,23 +103,23 @@ static int _gnrc_tcp_open(gnrc_tcp_tcb_t *tcb, const uint8_t *target_addr, uint1
     mutex_lock(&(tcb->function_lock));
 
     /* Connection is already connected: Return -EISCONN */
-    if (tcb->state != GNRC_TCP_FSM_STATE_CLOSED) {
+    if (tcb->state != FSM_STATE_CLOSED) {
```

Comment:
Since these Defines were moved into the internal Header, these defines should only be included during compilation of the internal GNRC TCPs Translation Units. This header should never be included in a Programm using TCP, therefore it doesn't matter.